### PR TITLE
Move some namespaces on Twig

### PR DESCRIPTION
**Pull request by loic425**
| Q               | A
| --------------- | -----
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | 
| License         | MIT

Introduced in 1.11

### CODE REVIEW

File: docs/responders.md [hunk 1]
```diff
@@ -40,9 +40,7 @@ As an example, we add a `foo` variable to the Twig template with `bar` as value.
 
 namespace App\Twig\Context\Factory;
 
-use phpDocumentor\Reflection\Types\Context;
-use Sylius\Component\Resource\Metadata\Operation;
-use Sylius\Component\Resource\Twig\Context\Factory\ContextFactoryInterface;
+use phpDocumentor\Reflection\Types\Context;use Sylius\Component\Resource\Metadata\Operation;use Sylius\Resource\Twig\Context\Factory\ContextFactoryInterface;
```
Comment (oallain) [inline docs/responders.md]: IMO, we should revert this change.

Comment (loic425) [reply docs/responders.md]: not really revert it, but it has to be fixed.

**Review by diimpp — APPROVED** [review]

**Review by lchrusciel — APPROVED** [review]

**Comment by lchrusciel** [comment]
Thank you, @loic425!
